doc/modules/cassandra/pages/auto-repair/overview.adoc

```diff
@@ -0,0 +1,125 @@
+= Overview of Auto Repair
+:navtitle: Auto Repair overview
+:description: Auto Repair concepts - How it works, how to configure it, and more.
+:keywords: CEP-37
+
+Repair orchestration inside Apache Cassandra. This fully Automated Repair scheduler inside Cassandra does not depend on the control plane, significantly reducing our operational overhead.
+At a high level, a dedicated thread pool is assigned to the repair scheduler. The repair scheduler inside Cassandra maintains a new replicated table under a distributed system_distributed keyspace. This table maintains the repair history for all the nodes, such as when it was repaired the last time, etc. The scheduler will pick the node(s) that run the repair first and continue orchestration to ensure Every table and all of their token ranges are repaired. The algorithm can also run repairs simultaneously on multiple nodes and splits the token range into subranges with the necessary retry to handle transient failures. The automatic repair runs as soon as we start a Cassandra cluster, like Compaction, and does not require human-in-the-loop.
+The scheduler supports Full, Incremental, and Preview repair types today with the following features. A new repair type, such as Paxos repair and any future repair types, should be extended with minimal dev work!
+
+=== Features
+- Capability to run on multiple nodes simultaneously
+- Default implementation and an interface to override the data set being repaired per repair session
+- Ability to extend the token split algorithms with the following two implementations readily available:
+- Splits token ranges by putting a max cap on the size of data being repaired as part of one repair session as well as a max cap at a table level - this is crucial for Incremental Repair (Default)
```

Comment:
```suggestion
- Splits token ranges by putting a cap on the size of data being repaired as part of one repair session as well as a max cap at a per-schedule level (Default)
```

Reply:
should also indent this (and the one below) as they go under the 'Ability' one above.

Reply:
Done